Lesson21/gokce/script.js

```diff
@@ -0,0 +1,125 @@
+/*
+1. Declare variable that store elements that we need to use for this task
+
+2. Add event listeners (on click - buttons , on change - inputs)
+
+3. Take data from current element and show it in the target output field.
+
+*/
+const dateInput = document.getElementById("date");
+const dateOutput = document.getElementById("selected-date");
+const timeslotButtons = document.querySelectorAll(".slot");
+const selectedTimeOutput = document.getElementById("selected-time");
+const form = document.getElementById("booking_form");
+const confirmation = document.getElementById("confirmation");
+const confirmedName = document.getElementById("confirmed-name");
+const confirmedEmail = document.getElementById("confirmed-email");
+const confirmedDate = document.getElementById("confirmed-date");
+const confirmedTime = document.getElementById("confirmed-time");
+const confirmButton = document.getElementById("confirm");
+const nameInput = document.getElementById("name");
+const nameOutput = document.getElementById("display-name");
+const hiddenPersonalInfoPart = document.getElementById("personal-div");
+const clientNamePart = document.getElementById("hidden-name-part");
+const emailInput = document.getElementById("email");
+const emailOutput = document.getElementById("display-email");
+
+// TODO: set min date for tomorrow or any other future date
+const today = new Date();
+// 1. Yarının Tarihini Hesaplama (Minimum Tarih)
+//    - Önce bugünün kopyasını alıyoruz (değişkenler karışmasın diye).
+const minDate = new Date(today);
+minDate.setDate(minDate.getDate() + 1); // Tarihi bir gün ileri taşı
```

Comment:
This will not always be true. If today is November 30, then the next date should be December 1, not November 31.